docs/vectors/sealedbox_vectors.rst

```diff
@@ -0,0 +1,84 @@
+SealedBox reference vectors
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Since libsodium's tests do not provide reference data for the SealedBox
+construction, the implementation is verified with a ``sealbox_test_vectors``
+utility program that produces and checks custom test vectors
+by making specific calls to libsodium API.
+
+To build the ``sealbox_test_vectors`` you need a ``C`` language compiler,
+a prebuilt libsodium library more recent than version 1.0.3 and the
+corresponding include headers.
+
+In a unix-like programming environment you shoud then execute:
+
+.. code-block:: bash
+
+    $ cc -o sealbox_test_vectors -l sodium sealbox_test_vectors.c
+
+The utility program can both generate list of vectors on standard
+output and check consistency of vectors pulled from standard input.
```

Comment:
I noticed running just `./sealbox_test_vectors` never exits as it waits for stdin. As piping is generally very fast, should the program have a timeout that prints some helpful message on how to use it if nothing is received from stdin?